coffee_cmd/src/main.rs

```diff
@@ -56,7 +56,29 @@ async fn main() -> Result<(), CoffeeError> {
             let remotes = coffee.list().await;
             coffee_term::show_list(remotes)
         }
-        CoffeeCommand::Upgrade => coffee.upgrade(&[""]).await,
+        CoffeeCommand::Upgrade { repo, all } => {
+            let result = match repo {
+                Some(repo) => coffee.upgrade(&repo, all).await,
+                None => coffee.upgrade("", all).await,
```

Comment:
```suggestion
                None => coffee.upgrade("", all).await,
```

this is the meaning of all? why we need to pass the `""` empty string? we can use the option here

Reply:
you can also remove the `all` because if the `repo` is none at this point the upgrade command will upgrade all the repository